src/bindings/python/src/openvino/__init__.py

```diff
@@ -72,6 +72,7 @@
 # Helper functions for openvino module
 from openvino.utils.data_helpers import tensor_from_file
 from openvino._ov_api import compile_model
+from openvino.frontend.pytorch.inlined_extension import inlined_extension
```

Comment:
I don't thin we need it here. This is fw specific

Reply:
Yeah, agree